doc/pac/Port Access Control.md

```diff
@@ -0,0 +1,844 @@
+
+
+
+**show mab <cr | interface\>**
+
+This command is used to show a summary of the global mab configuration and summary information of the mab configuration for all ports. This command also provides the detailed mab configuration for a specified port
+
+| Field         | Description                              |
+| ------------- | ---------------------------------------- |
+| Interface     | Given interface                          |
+| Admin Mode    | MAB admin mode on the given interface    |
+| MAB auth type | MAB authentication type (EAP_MD5, PAP, CHAP) |
+
+
+
+**show dot1x**
+
+This command is used to show a summary of the global 802.1x configuration.
+
+| Field               | Description                              |
+| ------------------- | ---------------------------------------- |
+| Administrative Mode | Indicates whether 802.1x is enabled or disabled. |
+
+
+
+
+show dot1x detail <all | interface\>
+
+This command is used to show details of 802.1x configuration on an interface.
+
+| Field            | Description                              |
+| ---------------- | ---------------------------------------- |
+| Interface        | Given Interface                          |
+| PAE Capabilities | The Port Access entity (PAE) functionality of this port. Possible values are Authenticator or None |
+
+
+
+### 3.7.4 Clear Commands
+
+**sonic-clear authentication sessions <interface <all | interface | mac \>\>**
+
+This command clears information for all Auth Manager sessions. All the authenticated clients are re-initialized and forced to authenticate again.
+
+
+
+# 4 Scalability
+
+The following scale is supported:
+
+| Configuration / Resource                 | Scale |
+| ---------------------------------------- | ----- |
+| Total number of authenticated clients on a port configured in Multiple Authentication host mode | 16    |
+| Total number of authenticated clients supported by the switch | 128   |
+
+
+
+# 5 Appendix: Sample configuration
+
+```
+config authentication port contol interface auto Ethernet10
+config authentication dot1x pae interface authenticator Ethernet10
+config authentication host-mode interface multi-auth Ethernet10
+config authentication interface max-users 10 Ethernet10
+config mab interface enable pap
+config dot1x system-auth-control enable
+config authentication periodic interface Ethernet10
+config authentication timer reauthenticate interface 600 Ethernet10
+```
+
+
+
+# 6 Future Enhancements
+
+1. Add configurability support for 802.1x and MAB timers.
+2. Add support for fallback VLANs like Guest, Unauth etc. VLAN. These are used to authorize clients if they fail authentication under various circumstances.
+3. Add support for RADIUS Authorization attributes like ACLs.
+4. Add support for multiple RADIUS servers.
```

Comment:
Please include the thoughts on supporting on SONIC-VS image.

Reply:
This is important as there are early validations with GNS3 before actual deployments.

Reply:
Yes this will be taken up in a future release.